src/main/kotlin/pet/project/app/dto/book/BookMapper.kt

```diff
@@ -0,0 +1,27 @@
+package pet.project.app.dto.book
+
+import pet.project.app.model.Book
+
+object BookMapper {
+
+    fun Book.toDto(): ResponseBookDto {
+        return ResponseBookDto(
+            this.id,
+            this.title,
+            this.description,
+            this.yearOfPublishing,
+            this.amazonPrice,
+            this.amountAvailable
+        )
+    }
```

Comment:
```suggestion
    fun Book.toDto() = ResponseBookDto(
            id,
            title,
            description,
            yearOfPublishing,
            amazonPrice,
            amountAvailable
    )
```
(formatting should be fixed in the suggestion)

1. The function is pretty simple and the return type is obvious. So in this case we can omit explicit return type
2. `this` is redundant

Reply:
Fixed: Updated as proposed